src/ovirtapi.js

```diff
@@ -402,14 +422,20 @@ OvirtApi = {
     const url = `${AppConfiguration.applicationContext}/api/vms/${vmId}`
     return OvirtApi._httpGet({ url })
   },
-  getVmsByPage ({ page }: { page: number }): Promise<Object> {
+  getVmsByPage ({ page, additional }: { page: number, additional: Array<string> }): Promise<Object> {
     OvirtApi._assertLogin({ methodName: 'getVmsByPage' })
-    const url = `${AppConfiguration.applicationContext}/api/vms/;max=${AppConfiguration.pageLimit}?search=SORTBY NAME ASC page ${page}`
+    let url = `${AppConfiguration.applicationContext}/api/vms/;max=${AppConfiguration.pageLimit}?search=SORTBY NAME ASC page ${page}`
+    if (additional && additional.length > 0) {
+      url += `&follow${additional.join(',')}`
```

Comment:
Does this really work? Don't we need `&follow=${additional` (`=` added)?

Same for line 437.